packages/design-picker/src/constants.ts

```diff
@@ -80,5 +80,5 @@ export const DEFAULT_ASSEMBLER_DESIGN = {
 export const FREE_THEME = 'free';
 export const PREMIUM_THEME = 'premium';
 export const DOT_ORG_THEME = 'dot-org';
-export const WOOCOMMERCE_THEME = 'woocommerce';
+export const BUNDLED_THEME = 'bundled';
```

Comment:
Is it a good name?

Reply:
Good question. I'd consider flipping it around so we're looking at `THEME_WITH_BUNDLED_SOFTWARE`, but that looks nothing like the other constants. 🤔

Reply:
Yeah! 🤔
I'll keep this one for now, but if any idea comes to your mind, feel free to let me know that I can change it. 😉